pkg/config/config.go

```diff
@@ -105,6 +108,9 @@ const (
 	DefaultServiceAccountConfigFile                 = "service-account-config.yml"
 	DefaultServiceAccountOutputsFile                = "service-account-outputs.json"
 
+	DefaultKubePrometheusStackValuesPath     = ""
```

Comment:
Empty string? Should probably be "./" if its supposed to represent the current dir. Empty string makes it look like we've missed something

Reply:
My bad, I did not notice that ya'll have dropped these in when making the PR, and the linter has been disabled for that section. Removed instead!